cvat/schema.yml

```diff
@@ -4576,6 +4576,15 @@ paths:
         description: A simple equality filter for the project_id field
         schema:
           type: integer
+      - name: resource
```

Comment:
Why is it called "resource", when it seems to correspond to the `target` field?

Reply:
Hm, probably I thought that using `resource` and `subresource` is better than `target` and `subresource`

Reply:
Well... it would've been better, had it been that way from the start. But the field is already called `target`, so it doesn't really make sense for the corresponding filter to be `resource`.